lib/fluent/parser.rb

```diff
@@ -155,18 +155,10 @@ def parse(text)
             when @time_key
               time = @mutex.synchronize { @time_parser.parse(value) }
               if @keep_time_key
-                record[name] = if @type_converters.nil?
-                                 value
-                               else
-                                 convert_type(name, value)
-                               end
+                record[name] = convert_type(name, value)
```

Comment:
Calling `convert_type` heavily is high cost.
So we should not call `convert_type` when user doesn't specify `types` parameter.